app/grandchallenge/components/templates/components/partials/ci_example_value.html

```diff
@@ -0,0 +1,36 @@
+{% load json%}
+
+<a href="#ciModal-{{ interface.pk }}"
+    data-toggle="modal"
+    data-target="#ciModal-{{ interface.pk }}"
+    class="text-decoration-none"
+    role="button"
+    title="View {{ interface.title }}"
+>
+    <div class="badge badge-primary">
+        <i
+                class="fas fa-fw fa-eye"
+        ></i>
+        View Example
+    </div>
+</a>
+
+<div class="modal" id="ciModal-{{ interface.pk }}" tabindex="-1" role="dialog"
+    aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered modal-lg" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title">
+                    {{ interface.title }} example
+                </h5>
+                <button type="button" class="close" data-dismiss="modal"
+                        aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="text-left modal-body">
+                <pre>{{ interface.example_value|json_dumps:4 }}</pre>
```

Comment:
If the `example_value` is `None`, and the `kind` is one of the JSON kinds then the default example for that interface should be provided. For now, duplicate the examples from the documentation in code.

Reply:
I see 20 examples in the code documentation for JSON kinds. Do you mean here duplicating them with an if statement in the template?

Reply:
It should not be in the template, do it in code. The template should look up a property on the model.

Reply:
Done